docs/src/interface_fluxes.md

```diff
@@ -5,41 +5,41 @@ To motivate this tutorial, we first note that `ClimaOcean`'s [`OceanSeaIceModel`
 1. Manage time-stepping multiple component models forward simultaneously,
 2. Compute and communicate fluxes between the component models.
 
-This tutorial therefore touches on one of the two main purposes of `OceanSeaIceModel`:
+This tutorial therefore touches on the latter of the two main purposes of `OceanSeaIceModel`:
 computing turbulent fluxes between model components.
 
 ## Component interfaces we consider
 
 The `OceanSeaIceModel` has atmosphere, ocean, and sea ice components (and will someday also have land and radiation components).
-This tutorial will hopefully eventually cover all turbulent flux computations, but for the time being we
+We envision that the tutorial will eventually cover all turbulent flux computations; for the time being we
 focus on atmosphere-ocean fluxes.
-Future expansions of this tutorial should cover atmosphere-sea-ice fluxes, ocean-sea-ice fluxes, ocean-land fluxes,
+Future expansions of this tutorial should cover atmosphere-sea ice fluxes, ocean-sea ice fluxes, ocean-land fluxes,
 and surface optical computations for radiation.
 
 ## Turbulent exchanges between the atmosphere and underlying media
 
-Exchanges of properties like momentum, heat, water vapor, and trace gases between the fluid atmosphere and its underlying surfaces ---
-ocean, sea ice, snow, land --- mediate the evolution of the Earth system.
+Exchanges of properties like momentum, heat, water vapor, and trace gases between the fluid atmosphere and its underlying surfaces --
+ocean, sea ice, snow, land -- mediate the evolution of the Earth system.
 Microscopic property exchange is mediated by a complex panolpy of processes including heat conduction, viscous and pressure form drag over rough surface elements, plunging breakers, and more.
-To represent atmosphere-surface exchanges, we construct a model of the near-surface atmosphere that connects a turbulent "similarity layer", usually a few meters thick, with a "constant flux layer"
-that buffers free atmospheric turbulence from microscopic surface exchange processes beneath.
+To represent atmosphere-surface exchanges, we construct a model of the near-surface atmosphere that connects a turbulent "similarity layer",
+which is usually a few meters thick, with a "constant flux layer"that buffers free atmospheric turbulence from microscopic surface exchange processes beneath.
 The problem of modeling property exchange then turns to the task of modeling turbulent atmospheric fluxes just above the constant flux layer.
 
 ## Bulk formula and similarity theory
 
 Within in each grid cell at horizontal position $x, y, t$, the atmosphere-surface
-turbulent fluxes of some quantity $\psi$ --- at the bottom of the similarity layer, and thus throughout
-the constant flux layer and across the surface --- is defined as
+turbulent fluxes of some quantity $\psi$ -- at the bottom of the similarity layer, and thus throughout
+the constant flux layer and across the surface -- is defined as
```

Comment:
markdown does not recognize `---` unfortunately so these were being rendered as a `--` next to a `-`.